pt/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,567 @@
 **1. Supervised Learning cheatsheet**
 
-&#10230;
+&#10230; Dicas de Aprendizado Supervisionado
 
 <br>
 
 **2. Introduction to Supervised Learning**
 
-&#10230;
+&#10230; Introdução ao Aprendizado Supervisionado
 
 <br>
 
 **3. Given a set of data points {x(1),...,x(m)} associated to a set of outcomes {y(1),...,y(m)}, we want to build a classifier that learns how to predict y from x.**
 
-&#10230;
+&#10230; Dado um conjunto de dados {x(1),...,x(m) associados a um conjunto de resultados {y(1),...,y(m)}, nós queremos construir um classificador que aprende como predizer y baseado em x.
 
 <br>
 
 **4. Type of prediction ― The different types of predictive models are summed up in the table below:**
 
-&#10230;
+&#10230; Tipos de predição - Os diferentes tipos de modelo de predição estão resumidos na tabela abaixo:
 
 <br>
 
 **5. [Regression, Classifier, Outcome, Examples]**
 
-&#10230;
+&#10230; [Regressão, Classificador, Resultado, Exemplos]
 
 <br>
 
 **6. [Continuous, Class, Linear regression, Logistic regression, SVM, Naive Bayes]**
 
-&#10230;
+&#10230; [Contínuo, Classe, Regressão Linear, Regressão Logística, Máquina de Vetores de Suporte, Naive Bayes]
```

Comment:
In  _Máquina de Vetores de Suporte_ would not it be the case to adopt the same pattern you used in _LWR - Locally Weighted Regression _?, i.e. putting the translation and the equivalent in English? It still has good materials in PT-BR which has the definition as SVM.

Reply:
I think it is not necessary because I've seen many materials in portuguese using the translated term. However, on the other I am not familiar, that's why I thought it would be better to keep both terms. What's your thoughts on this? @gabriel19913 and yours?

Reply:
@leportella I think in this case would be good to have the term in Portuguese and English. I am not familiar with it to know if the Portuguese tranlation, but if you are saying so, it would be nice to keep both.

Reply:
@leportella The oldest reference in PT-BR that I found was written by the PhD. [Ana Carolina Lorena](http://buscatextual.cnpq.br/buscatextual/visualizacv.do?id=K4777610P7), because of this that I thought that will sound awkward _Máquinas de Vetores de Suporte_. Actually, the only papers that I saw with this was in Econometrics tbh.